### DISCUSSION TITLE
German translation

### DISCUSSION THREAD

**Discussion by Klaus1189** [Xanashi/Icaros#61] [Show and tell]
I made german translation for installer and updated the german translation in Icaros itself:
https://drive.google.com/drive/folders/1jv19tOz4VQKBYPCaS1h_vJpLmglTm-X9?usp=sharing

If I did anything wrong please let me know and I fix it.
If something is updated, how can I check for updates or is it possible to get notified?

**Comment by Xanashi** [comment]
Sorry for the slow reply!
Thank you so much for updating the german translation, I really appreciate your work!
Could you please provide the .po files instead of the .pot files. (The .pot files doesn't actually contain the translations as they are the template files)

**Comment by Klaus1189** [comment]
I am back, I was out of my home for several days.
But if I compare the po and pot files in Notepad++ they match. So renaming the files could work, isn't it?
If it does not work, please let me know.

**Comment by Klaus1189** [comment]
And I take care of it :)

**Comment by Xanashi** [comment]
Ah, you're absolutely right, Klaus.
They just had the wrong file extension. 

Thank you for all your hard work. I'll make sure your translation is included in the upcoming release of Icaros!

**Reply by Klaus1189** [reply]
Thank you ❤️

**Reply by Klaus1189** [reply]
Can you check if any string is cut off?

**Reply by Xanashi** [reply]
Looks perfect on my end. Nothing is cut off and everything is translated. ❤️

**Comment by Klaus1189** [comment]
If you update anything, how can I know if there is anything new?
I want to keep with the translation :)

**Comment by Xanashi** [comment]
That's actually quite simple.

Whenever you download a new release of Icaros, you can just go into the _[icaros installation directory]\Resources\Localize_ folder,
and open the German translation PO file (Config.de.po) in PoEdit. 

You can then see all new strings that are currently missing a translation, 
as well as previous strings that maybe need work, due to changes.

**Reply by Klaus1189** [reply]
Thanks :)

**Reply by Xanashi** [reply]
Hi Klaus,
I've received a slightly different DE translation from another DE user, and was wondering if I could discuss
the different translation choices you made compared to the other translator?
If you don't mind and have the time, let me know. You should not feel obliged to do it though. ;)

**Reply by Klaus1189** [reply]
Hi,
Sorry for late reply. Can you show me the differences?

**Reply by Xanashi** [reply]
Hi Klaus, 
Sorry for the slow reply. Could you contact me at xanashi[at]gmail[dot]com?
I can send you the 2 po files for comparison.
